packages/scss/src/components/indexTableSorted/index.scss

```diff
@@ -0,0 +1,48 @@
+@use '@lucca-front/scss/src/components/tableSorted/exports' as tableSorted;
+@use 'exports' as *;
+
+.indexTable-head-row-cell {
+	&.mod-sortable,
+	.indexTable-head-row-cell-sortableButton {
+		@include tableSorted.sortable;
+
+		.indexTable.mod-S & {
+			@include tableSorted.sortableS;
+		}
+	}
+
+	.indexTable-head-row-cell-sortableButton:is(button) {
+		display: flex;
+		align-items: center;
+	}
+
+	&:is([aria-sort='descending']) {
+		@include descending;
```

Comment:
Cell padding is eliminated with this mixin.

Either it should not be done, or this padding should be used on the sort buttons.